inst/dev/install-deps.R

```diff
@@ -0,0 +1,26 @@
+options(repos = c(CRAN = "https://cloud.r-project.org"))
+if (getRversion() < "4.4.1") {
+  cli_alert_error("Please upgrade to R 4.4.1 or later.")
```

Comment:
```suggestion
  cli::cli_alert_error("Please upgrade to R 4.4.1 or later.")
```

Reply:
I was getting an error when running task install-with-pak that cli_alert_error can't be found.

Reply:
Perfect, thanks!